packages/client/package.json

```diff
@@ -7,24 +7,15 @@
     "type": "git",
     "url": "git://github.com/lens-protocol/lens-sdk.git"
   },
-  "type": "module",
-  "exports": {
-    ".": {
-      "types": "./dist/index.d.ts",
-      "default": "./dist/index.js"
-    },
-    "./package.json": "./package.json"
-  },
-  "types": "./dist/index.d.ts",
-  "main": "./dist/index.js",
+  "main": "dist/lens-protocol-client.cjs.js",
+  "module": "dist/lens-protocol-client.esm.js",
   "private": true,
   "sideEffects": false,
   "files": [
     "dist/**/*"
   ],
   "scripts": {
-    "build": "pnpm clean && tsup",
-    "clean": "rimraf ./dist",
```

Comment:
if you removed the `clean` script, do we still need `rimraf` as a dep here?

Reply:
no we don't, cheers